apiExamples/dynamic.html

```diff
@@ -0,0 +1,3 @@
+<auro-radio-group required id="dynamicExample">
+  <span slot="legend">Form label goes here</span>
+</auro-radio-group>
```

Comment:
Let's make sure that the documentation for this example explains that the "id" attribute must be added to all dynamically generated radio buttons.